Related2.java

```diff
@@ -0,0 +1,25 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Related2 {
+    public static boolean Realted(String first, String second) {
+        int result=0;
+        if(first.length()!=second.length())
+            return false;
+        for(int i=0;i<first.length();i++){
+            result+= first.charAt(i);
+        }
+        for(int i=0;i<second.length();i++){
+            result-= second.charAt(i);
+        }
+        return (result==0);
```

Comment:
This is not right, 
Take an example of `abc` and `ada`.
Your solution will result `true` but the answer should be `false`